torchtitan/components/tokenizer.py

```diff
@@ -82,8 +82,15 @@ def _load_config(self, config_path: str) -> Optional[dict]:
     def _load_tokenizer_from_path(self, tokenizer_path: str) -> Tokenizer:
         """Load tokenizer from various file formats."""
         if not os.path.exists(tokenizer_path):
-            raise FileNotFoundError(f"Tokenizer path '{tokenizer_path}' does not exist")
-
+            if "assets/tokenizer" in tokenizer_path:
+                raise FileNotFoundError(
+                    "Detected deprecated ./assets/tokenizer path. Remove --model.tokenizer_path "
+                    "and download to --model.hf_assets_path using ./scripts/download_hf_assets.py"
```

Comment:
I suggest we add link https://github.com/pytorch/torchtitan/tree/main/torchtitan/models/deepseek_v3#download-tokenizer for better user guidance.